.github/workflows/swift_package_test.yml

```diff
@@ -1,8 +1,24 @@
-name: Swift Linux Matrix
+name: Swift Matrix
 
 on:
   workflow_call:
     inputs:
+      macos_xcode_versions:
+        type: string
+        description: "Exclude Xcode version list (JSON)"
+        default: "[\"16.0\", \"16.1\", \"16.2\"]"
+      macos_exclude_xcode_versions:
+        type: string
+        description: "Exclude Xcode version list (JSON)"
+        default: "[{\"xcode_version\": \"\"}]"
+      macos_versions:
+        type: string
+        description: "macOS version list (JSON)"
+        default: "[\"sequoia\"]"
+      macos_archs:
+        type: string
+        description: "macOS arch list (JSON)"
+        default: "[\"ARM64\"]"
```

Comment:
Same comment as with the macOS version, I would add a comment like: `This requires a self-hosted macOS runner with this architecture to be set up.`